doc/reference/reference_lua/fio.rst

```diff
@@ -197,6 +197,10 @@ Below is a list of all ``fio`` functions and members.
     Given a full path name, remove all but the final part (the file name).
     Also remove the suffix, if it is passed.
 
+    Note that the basename of a path with the trailing slash is the empty
```

Comment:
```suggestion
    Note that the basename of a path with a trailing slash is an empty
```

Reply:
I guess I should also replace

> Example with the trailing slash:

with

> Example with a trailing slash:

Reply:
Updated.